src/main/java/seedu/duke/command/AddCommand.java

```diff
@@ -81,6 +82,11 @@ public void execute(TransactionList transactions, Ui ui, Storage storage) throws
         // Throws exception if there are no parameters
         checkParameterExist(splits);
 
+        assert (splits.length == TAG_LIMIT) : "Number of split inputs must equal 5.";
+        // At this stage, the number of split inputs will be == 5 as all the 5 tags must exist
+        // Any additional tags would throw an exception before this assertion.
```

Comment:
Good assertion over here, note that for single-line comments there will be no period at the end ;)